src/Compiler/SyntaxTree/PrettyNaming.fs

```diff
@@ -224,6 +224,7 @@ let keywordsWithDescription: (string * string) list =
         "rec", FSComp.SR.keywordDescriptionRec ()
         "return", FSComp.SR.keywordDescriptionReturn ()
         "return!", FSComp.SR.keywordDescriptionReturnBang ()
+        "sig", FSComp.SR.keywordDescriptionSig ()
```

Comment:
Are you sure this is the right place? There are more of them in the [keyword reference](https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/keyword-reference) and all the others are a bit elsewhere in this file.

(and yeah I wonder how this particular one got slipped)

Reply:
This is where the API that does the escaping look for the keywords.

Reply:
> There are more of them in the [keyword reference](https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/keyword-reference) and all the others are a bit elsewhere in this file.

Hmm, maybe we should do a systematic update of that list then

Reply:
@KevinRansom @vzarytovskii any thoughts on this? I'm not very strong at this part here, but it feels inconsistent here now.